src/monster860/fastdmm/FastDMM.java

```diff
@@ -482,27 +482,33 @@ else if(dwheel < 0)
 		
 		while(Mouse.next()) {
 			if(Mouse.getEventButtonState()) {
+				try {
+					currPopup.setVisible(false);
+				} catch (NullPointerException e) {
+					// Doesn't exist. Oh well.
+				}
```

Comment:
That changes the behavior. Doing this causes it to disappear correctly the first time, but then it no longer creates the context menu on right click.